### DISCUSSION TITLE
波形应该怎么用

### DISCUSSION THREAD

**Discussion by eatgrass** [ZuodaoTech/everyone-can-use-english#797] [General]
到目前为止，我还不知道如何借助音频波形来帮助学习，我甚至比较怀疑它是否真的有必要，毕竟enjoy花了大量性能在波形的读取生成上，而且这个功能是强制而非可选的，并为此折中牺牲了一些在我看来更有必要的能力，例如 #720
可能是我对这个功能的认识不足，想在此听听开发者和其他使用者的想法。

**Comment by an-lee** [comment]
可以参考*韵律*一节： https://1000h.org/sounds-of-american-english/2.5.1-pitch.html

**Comment by tan2meng** [comment]
波形可以直观“看到”单词们的时值，音高，连接，停顿状态。而不是单靠听觉“听”一个维度了解句子发音要素。
